### PR TITLE
feat: Clean up LoginPage.xaml

### PR DESCRIPTION
**Pull request by MaximeDion-Work**
GitHub Issue: #

## PR Type

What kind of change does this PR introduce?

- Feature

## Description

General clean up of the xaml file that include stuff like: 
Added comments to organize code
Reorganize the order of properties in control in a more logical manner 
Remove unecessary blend using
Ensure proper use of material color ressources
Reorganization of layout (like using stack panels instead of mannually defines rows)
Make sure Margin and Padding are set using multiples of 4 (When it does not disturb design)

## PR Checklist 
Please check if your PR fulfills the following requirements:

- [ ] ~~Interface members are XML documented~~
- [ ] ~~Documentation (XML or comments) has been added and/or existing documentation has been updated~~
- [ ] ~~[Architecture documents](./doc/Architecture.md) have been updated~~
- [x] Tested on all relevant platforms

## Internal Issue (If applicable):

Clean up LoginPage.xaml
https://dev.azure.com/nventive/Practice%20committees/_workitems/edit/259216

### CODE REVIEW

File: src/app/ApplicationTemplate.UWP/Views/Content/Authentication/LoginPage.xaml [hunk 1]
```diff
@@ -87,39 +84,44 @@
...
 						Content="CREATE ACCOUNT"
 						Command="{Binding NavigateToCreateAccountPage}"
 						Style="{StaticResource MaterialOutlinedButtonStyle}"
 						HorizontalAlignment="Stretch"
-						Margin="42,0" />
+						Margin="0,16,0,0" />
 
-				<Rectangle Margin="42,32"
+				<!-- Separator -->
+				<Rectangle Fill="{StaticResource MaterialPrimaryFocusedBrush}"
```
[the remark this answers is not in the document]
Comment (jeanplevesque) [reply src/app/ApplicationTemplate.UWP/Views/Content/Authentication/LoginPage.xaml:113]: not fixed

File: src/app/ApplicationTemplate.UWP/Views/Content/Authentication/LoginPage.xaml [hunk 2]
```diff
@@ -34,40 +34,38 @@
...
-			   Stretch="UniformToFill"
-			   HorizontalAlignment="Stretch"
-			   VerticalAlignment="Stretch" />
+			   Stretch="UniformToFill" />
 
 		<ScrollViewer>
 			<StackPanel toolkit:VisibleBoundsPadding.PaddingMask="Top,Bottom"
-						VerticalAlignment="Center">
+						VerticalAlignment="Center"
+						Margin="44,32">
```
[the remark this answers is not in the document]
Comment (jeanplevesque) [reply src/app/ApplicationTemplate.UWP/Views/Content/Authentication/LoginPage.xaml]: You changed 44 to 40. I don't feel like it answers the question.

File: src/app/ApplicationTemplate.UWP/Views/Content/Authentication/LoginPage.xaml [hunk 3]
```diff
@@ -87,39 +84,44 @@
...
-						Margin="42,0" />
+						Margin="0,16,0,0" />
 
-				<Rectangle Margin="42,32"
+				<!-- Separator -->
+				<Rectangle Fill="{StaticResource SeparatorBrush}"
 						   Height="1"
 						   HorizontalAlignment="Stretch"
-						   Fill="{StaticResource MaterialOnSurfacePressedBrush}" />
+						   Margin="0,32,0,0"/>
```
Comment (carlh98) [inline src/app/ApplicationTemplate.UWP/Views/Content/Authentication/LoginPage.xaml:116]: spacing

File: src/app/ApplicationTemplate.UWP/Views/Content/Authentication/LoginPage.xaml [hunk 4]
```diff
@@ -87,39 +84,44 @@
 								 ue:PasswordBoxBehavior.DismissKeyboardOnEnter="True" />
 				</local:DataValidationView>
 
+				<!-- Forgot Password Button -->
 				<HyperlinkButton Content="Forgot password ?"
 								 x:Uid="Login_ForgotPassword"
 								 Command="{Binding NavigateToForgotPasswordPage}"
-								 Foreground="{StaticResource MaterialOnSurfaceMediumBrush}"                                                                                                                                                                                                                                                                      
-								 Margin="42,0" />
+								 Foreground="{StaticResource MaterialPrimaryMediumBrush}"
```
Comment (carlh98) [inline src/app/ApplicationTemplate.UWP/Views/Content/Authentication/LoginPage.xaml:91]: Should it not be MaterialOn since it's a Foreground

**Pull request merged**